src/main/java/com/CourseScheduleBuilder/controller/RegistrationController.java

```diff
@@ -0,0 +1,45 @@
+package com.CourseScheduleBuilder.controller;
+
+
+import com.CourseScheduleBuilder.Model.User;
+import com.CourseScheduleBuilder.Repositories.UserRepo;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+/**
+ */
+@RestController
+public class RegistrationController {
+
+    @Autowired //needed to point to the same repo database
+    private UserRepo userrepo;
+
+    @PostMapping("/registration")
+    @CrossOrigin
+    @ResponseBody
+    public boolean login(@RequestBody User user){
+
+        if (userrepo.findByEmail(user.getEmail()) == null ){
+            userrepo.save(user);
+             return true;
+        }
+            else {
+                System.out.println("DUPLICATE ACCOUNT CREATION ATTEMPTED");
+                System.out.println(user.getEmail());
```

Comment:
We should start using a logger s4j for logging application info. More on that later